debian/logitechmediaserver.default

```diff
@@ -1,4 +1,10 @@
-# User to run Logitech Media Server as
+# User to run Logitech Media Server as.
+# Only applicable to platforms that do not use systemd.
+# Running the server as a user other than the default squeezeboxserver
```

Comment:
Any reason why you'd stick with `squeezeboxserver` for the user but use `logitechmediaserver` in most other cases?

Reply:
I've chosen to stick with what the existing installer does, this proposed change only adds a systemd unit file to the existing installation.

The existing installation is somewhat "conflicted". It creates the user `squeezeboxserver`, and sets up directories under a `squeezeboxserver` hierarchy. (`/var/lib/squeezeboxserver`, `/var/log/squeezeboxserver`).

`logitechmediaserver` is used only (I think) as the service name (i.e. the init script is named `/etc/init.d/logitechmediaserver`) and some items in the `/etc` hierarchy use that name. (So we have `/etc/default/logitechmediaserver`, `/etc/logrotate.d/logitechmediaserver`, but `/etc/squeezeboxserver`).

The systemd unit file that this change installs is named `logitechmediaserver.service`. Keeping the service name consistent between SysV and systemd is important, otherwise `systemd` will launch two instances - once from the unit file and one from the SysV init script. 

A tidying up exercise might be nice to do, but I wouldn't wish to incorporate it into this change.

Reply:
I'd agree that keeping things as similar as possible to the current set up is the way to go. At the expense of maintaining the current inconsistency in naming.